packages/common/src/adapters/access.ts

```diff
@@ -34,3 +39,27 @@ export const accessConditionsFromSDK = (
     throw new Error(`Unsupported access gate type: ${input}`)
   }
 }
+export const accessConditionsToSDK = (
+  input: AccessConditions
+): TrackMetadata['downloadConditions'] => {
+  if ('follow_user_id' in input) {
+    return {
+      followUserId: encodeHashId(input.follow_user_id)
+    }
+  } else if ('nft_collection' in input) {
```

Comment:
I don't know if these keys are default initialized to undefined or not. But should probably check that the key has a usable value as well.

Reply:
Going to use the typeguards we already have!